python/python/glide/async_commands/core.py

```diff
@@ -1527,6 +1527,26 @@ async def sismember(
             await self._execute_command(RequestType.SIsMember, [key, member]),
         )
 
+    async def sinter(self, keys: List[str]) -> Set[str]:
+        """
+        Gets the intersection of all the given sets.
+        See https://redis.io/docs/latest/commands/sinter/ for more details.
+
+        Args:
```

Comment:
Add same-slot notice (see #282 for example)
> When in Cluster mode, all keys must map to the same hash slot.

Reply:
Will do.